internal/utils/testutil/cli_args.go

```diff
@@ -14,6 +14,13 @@
 
 package testutil
 
+import "time"
+
+// Available const with Timeout
```

Comment:
Change this, for more descriptive text.

"Average timeout for analysis in the example directory."